pom.xml

```diff
@@ -52,6 +52,13 @@
   </issueManagement>
   <mailingLists>
     <mailingList>
+      <name>DistributedLog User</name>
+      <subscribe>user-subscribe@distributedlog.incubator.apache.org</subscribe>
+      <unsubscribe>user-unsubscribe@distributedlog.incubator.apache.org</unsubscribe>
+      <post>user@distributedlog.incubator.apache.org</post>
+      <archive>http://www.mail-archive.com/user%distributedlog.incubator.apache.org</archive>
```

Comment:
I don't think we need to encode the @ in the xml. I updated the pom file to use 'user@distributedlog' now.